lib/media/streaming_engine.js

```diff
@@ -1004,7 +1004,7 @@ shaka.media.StreamingEngine = class {
       // Do not try to predict the next update.  Just poll twice every second.
```

Comment:
The comment "Just poll twice every second." need to be changed.

Reply:
Done!